src/microbe_stage/editor/MicrobeEditorReportComponent.tscn

```diff
@@ -73,9 +73,6 @@ anchor_bottom = 1.0
 margin_top = -0.200001
 theme = ExtResource( 9 )
 script = ExtResource( 11 )
-__meta__ = {
```

Comment:
Random changes to unrelated files that aren't related to the PR at all should be reverted.